tests/ui/regex.stderr

```diff
@@ -29,7 +29,10 @@ error: regex syntax error: invalid character class range, the start must be <= t
 LL |     let some_unicode = Regex::new("[é-è]");
    |                                     ^^^
 
-error: regex syntax error on position 0: unclosed group
+error: regex parse error:
+           (
```

Comment:
Could we extend the span to cover the quotation marks? The way it looks now throws me off a bit.

Reply:
I don't think we can, it's the error as printed by the regex crate which doesn't know of the source representation

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=0193cd9f840f62b49211c23e647ae5e0

I'd say it looks slightly better in a terminal where it's grouped together by being bolded, but yeah the formatting is a little out of place

![image](https://user-images.githubusercontent.com/1830331/215329040-026d907d-707e-4f85-8633-f8f8430cc9bb.png)

Reply:
I see. Ok then.